CHANGELOG.md

```diff
@@ -364,7 +364,7 @@
 
 * [CHANGE] `mimir-continuous-test` has been deprecated and replaced by a Mimir module that can be run as a target from the `mimir` binary using `mimir -target=continuous-test`. #7753
 * [CHANGE] `-server.metrics-port` flag is no longer available for use in the module run of mimir-continuous-test, including the grafana/mimir-continuous-test Docker image which uses the new module. Configuring this port is still possible in the binary, which is deprecated. #7747
-* [CHANGE] Allowed authenticatication to Mimir using both Tenant ID and basic/bearer auth #7619.
+* [CHANGE] Allowed authenticatication to Mimir using both Tenant ID and basic/bearer auth. #7619 #9038
```

Comment:
Can you move this (copying the entry if you'd like) to the `main / unreleased` section since this isn't going to be in 2.13?